iree/compiler/Dialect/Flow/Transforms/DispatchLinalgOnTensors.cpp

```diff
@@ -443,6 +523,18 @@ static void tryToTieOperandsAndResults(
   }
 }
 
+void replaceAllUsesWithinDispatchOp(IREE::Flow::DispatchWorkgroupsOp dispatchOp,
```

Comment:
I think this is outside anonymous namespace since other methods are explicitly static. Please also add `static` to this method.